websites/G/Granblue Fantasy/presence.ts

```diff
@@ -23,13 +77,30 @@ presence.on("UpdateData", async () => {
   } else if (href.includes("/#result"))
     presenceData.details = "In a Quest result screen";
   else if (href.includes("/#raid") || href.includes("/#raid_multi")) {
+    if (health === 0) {
+      presenceData.state = `At ${
+        document.getElementsByClassName(
+          "btn-enemy-gauge prt-enemy-percent alive"
+        )[0].textContent
+      }`;
+    } else if (health === 1) {
+      const boss = gameStatus.boss.param.find(x => x.alive),
+        hp = parseInt(boss.hp);
+      presenceData.state = `${hp.toLocaleString()} [${(
+        (100 * hp) /
+        parseInt(boss.hpmax)
+      ).toLocaleString(void 0, {
+        maximumFractionDigits: 2
+      })}%]`;
+    }
     presenceData.details =
       document.getElementsByClassName("name")[0].textContent;
-    presenceData.state = `At ${
-      document.getElementsByClassName(
-        "btn-enemy-gauge prt-enemy-percent alive"
-      )[0].textContent
-    }`;
+    if (djeeta) {
+      const charaAlive =
```

Comment:
~~Me and my ignorance~~ Nothing, will fix ASAP

Reply:
Done

Reply:
![image](https://user-images.githubusercontent.com/77577746/149684461-c352ad95-006a-4f47-9f96-a40a82542777.png)
Apparently it would break at compiling but that is fixed now